### PR TITLE
Add a little bit of tests

### PR DESCRIPTION
**Pull request by adentong**
Several changes here, all very minor, hence why I grouped them together:
1. Remove getProductMaxQty, which was recently added, but then immediately became deprecated
2. Remove a little bit of code in ApiController since $quote isn't even defined
3. Rename BoltGlobalTrait to BoltGlobalTraitTest, as is our convention
4. Added a test to WebhookTraitTest, though @mccottry please comment if you think this is a silly test and should be removed.
5. Reformatting OrderCreationExceptionTest to conform to our standards, removed a test that's covering things already covered in another test, and added a new test to cover one missing line

#changelog Add some unit tests

# Type of change

- [ ] Bug fix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] Test

# How Has This Been Tested?
Please validate that you have tested your change in at least one of the following areas:

- [x] Successfully tested locally (or docker image)
- [x] Successfully tested on a staging or sandbox server
- [ ] Successfully tested on a merchant's staging server

# Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] New and existing unit tests pass locally with my changes.
- [x] I have created or modified unit tests to sufficiently cover my changes.
- [ ] I have added my Asana task link and provided a changelog message if applicable.

### CODE REVIEW

**Comment by adentong** [comment]
This is ready for review.

File: tests/unit/testsuite/Bolt/Boltpay/OrderCreationExceptionTest.php [hunk 1]
```diff
@@ -9,9 +9,10 @@
 class Bolt_Boltpay_OrderCreationExceptionTest extends PHPUnit_Framework_TestCase
 {
     /**
-     * Test JSON response for general error with default parameters
+     * @test
+     * JSON response for general error with default parameters
      */
-    public function testGeneralExceptionJson()
+    public function initializeGeneralException_setsCorrectJsonAndHttpCode()
```
Comment (mccottry) [inline tests/unit/testsuite/Bolt/Boltpay/OrderCreationExceptionTest.php]: @adentong 
for all test methods, the format should be as follows:

<methodTested>_<conditionUnderWhichTestIsRun>_<expectedOutcome>

Comment (adentong) [reply tests/unit/testsuite/Bolt/Boltpay/OrderCreationExceptionTest.php]: Fair point. Let me update.

**Comment by adentong** [comment]
Looking to merge this in before the sprint ends tomorrow. @mccottry Do you think you could take a look again?

**Pull request merged**